src/Tribe/Admin/Filter_Bar/Provider.php

```diff
@@ -0,0 +1,143 @@
+<?php
+namespace Tribe\Events\Admin\Filter_Bar;
+
+use Tribe__Events__Main;
+use Tribe__Settings_Tab;
+use Tribe__Admin__Helpers;
+
+// Bail if Filter Bar is already installed.
+if ( ! class_exists( 'Tribe__Events__Filterbar__View' ) ) {
```

Comment:
This class exists will cause fatale, we should have this conditional on the register method, so that if this class doesn’t exist it doesn’t continue to do the rest

Reply:
@bordoni I've moved the conditional to the register method.